openpype/hosts/maya/api/lib.py

```diff
@@ -1159,23 +1227,38 @@ def extract_alembic(file,
         # Ensure valid types are converted to frame range
         assert isinstance(startFrame, _alembic_options["startFrame"])
         assert isinstance(endFrame, _alembic_options["endFrame"])
-        kwargs["frameRange"] = "{0} {1}".format(startFrame, endFrame)
+        frameRange = "{0} {1}".format(startFrame, endFrame)
     else:
         # Allow conversion from tuple for `frameRange`
-        frame_range = kwargs["frameRange"]
-        if isinstance(frame_range, (list, tuple)):
-            assert len(frame_range) == 2
-            kwargs["frameRange"] = "{0} {1}".format(frame_range[0],
-                                                    frame_range[1])
+        if isinstance(frameRange, (list, tuple)):
+            assert len(frameRange) == 2
+            frameRange = "{0} {1}".format(frameRange[0], frameRange[1])
```

Comment:
Should we be then setting `startFrame` and `endFrame` here from the `tupel` or `list`?

Reply:
Could you elaborate?
`frameRange` is what is used in the end. The arguments `startFrame` and `endFrame` is present are converted to the `frameRange` string.

Reply:
Ah we don't really need those two later on, we just care about the `frameRange` itself; so ignore what I said.